site/content/blogs/release-v130.en.md

```diff
@@ -38,6 +38,16 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
 
 ## Ctrl-C
 
+When your Copilot Service is failing to deploy, You can now hit `Ctrl-C` to rollback your current CloudFormation stack to previous configuration.
+If you hit 'Ctrl-C' again during a rollback, the rollback of CloudFormation stack will continue in the background.
```

Comment:
```suggestion
If you hit 'Ctrl-C' a second time, the rollback of CloudFormation stack will continue in the background.
```

Reply:
piggybacking on danny's suggestion - 
```
If you hit 'Ctrl-C' a second time, the program will exit, but the stack rollback will continue.
```

"in the background" sort of sound to me that, the program `copilot` is still running in the background, just detached from the terminal; whereas in reality really the program is stopped.